### PR TITLE
Add option to display only solutions selected with control+click

### LINKED ISSUE #442: Option to show only designs selected with control+click in design browser

### PR DESCRIPTION
**Pull request by AlexHalb**
Resolves #442

### CODE REVIEW

**Comment by luxaritas** [comment]
I don't think this should be a setting - it doesn't make sense for this to persist across different puzzles.

File: src/eterna/mode/DesignBrowser/DesignBrowserMode.ts [hunk 1]
```diff
@@ -92,6 +92,7 @@ export default class DesignBrowserMode extends GameMode {
         this._wholeRowWidth = 0;
         this._voteProcessor = new VoteProcessor(puzzle.maxVotes);
         this._initialSolution = initialSolution;
+        this._onlySelectedVisible = Eterna.settings.designBrowserOnlySelectedVisible.value;
```
Comment (luxaritas) [inline src/eterna/mode/DesignBrowser/DesignBrowserMode.ts:95]: Lingering setting

**Comment by luxaritas** [comment]
Wrapping around using the next/prev buttons while in the design browser seems to be broken - I think it's a regression with this PR

**Comment by AlexHalb** [comment]
I think the issue might be that the hotkeys aren't working - should be resolved by #470

**Pull request merged**